lib/ads/ad_manager.js

```diff
@@ -161,8 +161,20 @@ shaka.ads.AdManager = class extends shaka.util.FakeEventTarget {
     this.ssAdManager_ = null;
     /** @private {shaka.ads.AdsStats} */
     this.stats_ = new shaka.ads.AdsStats();
+    /** @private {string} locale */
+    this.locale_ = '';
```

Comment:
Let's set this to navigator.language by default.

Reply:
I changed it!